client/core/core.go

```diff
@@ -3192,8 +3198,21 @@ func (c *Core) RescanWallet(assetID uint32, force bool) error {
 			assetID, unbip(assetID), err)
 	}
 
+	walletDef, err := asset.WalletDef(assetID, wallet.walletType)
+	if err != nil {
+		return newError(assetSupportErr, "asset.WalletDef error: %w", err)
+	}
+
+	var bday uint64 // unix time seconds
+	if walletDef.Seeded {
+		creds := c.creds()
+		if !creds.Birthday.IsZero() {
+			bday = uint64(creds.Birthday.Unix())
+		}
+	}
+
 	// Begin potentially asynchronous wallet rescan operation.
-	if err = wallet.rescan(c.ctx); err != nil {
+	if err = wallet.rescan(c.ctx, bday); err != nil {
```

Comment:
Why do we need to pass the birthday here if it's already provided in `CreateWallet`? I see you used it in dcr but not btc.